registry/insecure.md

```diff
@@ -108,3 +108,22 @@ $ update-ca-trust enable
 ```
 
 Restart Docker for the  changes to take effect.
+
+### Windows
+
+In your browser navigate to `https://<dtr-url>/ca` to download the TLS
```

Comment:
Docker Registry doesn't have this functionality, so in the previous step of this tutorial you are creating the TLS certificates by yourself. Can you rephrase this to say something like:

"Configure Windows to trust your certificate. Open Windows Explorer ..."